.gitignore

```diff
@@ -24,6 +24,7 @@ cl/assets/bundles
 cl/assets/media/bulk-data/
 cl/assets/media/financial-disclosures/
 cl/assets/media/harvard_corpus/
+cl/assets/media/case.law/
```

Comment:
We dont want to store this separately from the `harvard_corpus` if thats possible